src/components/Checkout/DonationNudge/DonationNudgeBoxes.vue

```diff
@@ -55,6 +54,9 @@
 								ref="customDonationInputDesktop"
 								name="customDonationInputTextDesktop"
 								maxlength="10"
+								tabindex="10"
+								id="customDonationInput"
+								@click.capture="formatAndSubmitOnEnter"
 								class="nudge-box-input nudge-box-input-desktop"
```

Comment:
The event and value for this input would be better handled with v-model assigned to a data item and @click.enter with your method call.

Reply:
if this would require a significant refactor of the component you can skip for now...